mm2src/coins/z_coin.rs

```diff
@@ -696,7 +710,7 @@ pub async fn z_coin_from_conf_and_params(
     ticker: &str,
     conf: &Json,
     params: &ZcoinActivationParams,
-    consensus_params: ZcoinConsensusParams,
+    consensus_params: ZcoinProtocolInfo,
```

Comment:
shouldn't `consensus_params` be renamed to `protocol_info`? `z_coin_from_conf_and_params` fn

Reply:
Yes, it should :slightly_smiling_face: Will do tomorrow.